devdoc/amqp_management_requirements.md

```diff
@@ -180,17 +180,17 @@ int amqp_management_execute_operation_async(AMQP_MANAGEMENT_HANDLE amqp_manageme
 
 **SRS_AMQP_MANAGEMENT_01_055: [** `amqp_management_execute_operation_async` shall start an AMQP management operation. **]**
 
-**SRS_AMQP_MANAGEMENT_01_056: [** On success it shall return 0. **]**
+**SRS_AMQP_MANAGEMENT_01_056: [** On success it shall return an ASYNC_OPERATION_HANDLE. **]**
```

Comment:
>ASYNC_OPERATION_HANDLE [](start = 64, length = 22)

Consider enclosing in backticks `. Looks nicer in .md - something we're doing nowadays.

Reply:
Very true! Fixed.